### PR TITLE
Add: Amphibious Transport Voice Lines From Red Alert 2

### PR DESCRIPTION
**Pull request by commy2**
As suggested.

HOWEVER

This does need localization. And I'm not sure RA exists for all languages that ZH supports. Also, I do not have any files besides EN and DE in any case.

### CODE REVIEW

**Comment by xezon** [comment]
Nice. In what way does it need localization? Do you mean tool tip for the unit? What are these original sounds? Unrelated unit voices?

**Comment by commy2** [comment]
She speaks English. We'd need international voice lines.

**Comment by xezon** [comment]
Oh right. That will be difficult to achieve I expect. However, we would only need German and French, because all other languages are English Spoken Language.

**Comment by xezon** [comment]
You write you have German + English. So we would only need French on top. Any idea where to source that from?

**Comment by commy2** [comment]
From someone that has Red Alert 2 in French I suppose.

File: Patch104pZH/GameFilesEdited/Data/INI/Voice.ini [hunk 1]
```diff
@@ -5621,41 +5621,33 @@ End
 ;===================================================
 
 AudioEvent AmphibiousTransportVoiceSelect
-  Sounds = vchisea vchiseb vchisec vchised vchisee vchisef
+  Sounds = vampsea vampseb vampsec vampsed ;vampsee
```
Comment (xezon) [inline Patch104pZH/GameFilesEdited/Data/INI/Voice.ini:5624]: I had to listen to **vampsee** to understand why you commented it :-)

**Review by xezon — APPROVED** [review]
Change is ok as is. This means all languages will have English audio for now. This is ok for all languages except German and French. Can make a second pass to add English and German.

**Squash when commit.**

**Pull request merged**